administration/src/bp-modules/NavigationBar.tsx

```diff
@@ -69,7 +69,7 @@ const Navigation = ({ onSignOut }: Props): ReactElement => {
             <Button minimal icon='path-search' text='Region verwalten' />
           </NavLink>
         ) : null}
-        {role === Role.ProjectAdmin && config.userImportApiEnabled ? (
+        {(role === Role.ProjectAdmin && config.userImportApiEnabled) || role === Role.ExternalVerifiedApiUser ? (
```

Comment:
:wrench:
I think it would improve the readability if we introduce a variable like
`canSeeProjectSettings` or sth similar